boards/variscite/imx93_var_dart/doc/index.rst

```diff
@@ -0,0 +1,214 @@
+Overview
+********
+
+The DART-MX93 offers a high-performance processing for a low-power System-on-Module.
+The product is based on the i.MX 93 family which represents NXP’s latest power-optimized
+processors for smart home, building control, contactless HMI, IoT edge, and Industrial
+applications.
+
+The i.MX 93 includes powerful dual Arm® Cortex®-A55 processors with speeds up to 1.7 GHz
+integrated with a NPU that accelerates machine learning inference. A general-purpose Arm®
+Cortex®-M33 running up to 250 MHz is for real-time and low-power processing. Robust control
+networks are possible via CAN-FD interface. Also, dual 1 Gbps Ethernet controllers, one
+supporting Time Sensitive Networking (TSN), drive gateway applications with low latency.
+
+Zephyr OS is ported to run on either the Cortex®-A55 or the Cortex®-M33.
+
+Specs Summary
+*************
+
+  - CPU
+
+    - NXP i.MX 93:
+    - 2x Cortex®-A55 @ 1.7GHz
+    - 1x Cortex®-M33 @ 250 MHz
+    - 1x Ethos-U65 microNPU 0.5 TOPS
+  - Memory
+
+    - Up to 2GB LPDDR4 RAM
+  - GPU
+
+    - PXP 2D Pixel acceleration engine
+  - NPU (Neural Processing Unit)
+
+    - Neural Network performance (256 MACs operating up to 1.0 GHz and 2 OPS/MAC)
+    - NPU targets 8-bit and 16-bit integer RNN
+    - Handles 8-bit weights
+  - Display
+
+    - LVDS up to 1366x768p60 or 1280x800p60
+    - Parallel RGB up to 1366x768p60 or 1280x800p60
+    - 1x MIPI DSI up to 1920x1200p60 24-bit
+  - Network
+
+    - 2x 10/100/1000 Mbit/s Ethernet Interface
+    - Certified Wi-Fi 802.11ax/ac/a/b/g/n
+    - Bluetooth/BLE 5.4
+  - Camera
+
+    - One 2-lane MIPI CSI-2 camera input
+  - Audio
+
+    - Headphones
+    - Microphone: Digital, Analog (stereo)
+    - 3x I2S(SAI), S/PDIF, PDM 4CH
+  - USB
+
+    - 2x USB 2.0 OTG
+  - Serial interfaces
+
+    - SPI: x7
+    - I2C: x7
+    - UART: x7, up to 5 Mbps
+    - CAN: x2
+  - Temperature range
+
+    - -40°C to 85°C
+
+More information about the SoM can be found at the
+`Variscite Wiki`_ and
+`Variscite website`_.
+
+Supported Features
+******************
+
+.. zephyr:board-supported-hw::
+
+.. note::
+
+   It is recommended to disable peripherals used by the M33 core on the Linux host.
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 24 MHz.
+Cortex-A55 Core runs up to 1.7 GHz.
+Cortex-M33 Core runs up to 200MHz in which SYSTICK runs on same frequency.
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+CPU's UART7 for A55 core and M33 core.
+
+Programming and Debugging (A55)
+*******************************
+
+Copy the compiled ``zephyr.bin`` to the boot directory of the SD card and
+plug the SD card into the board. Power it up and stop the U-Boot execution at
+prompt.
+
+Use U-Boot to load and run zephyr.bin on the Cortex-A55:
+
+.. code-block:: console
+
+   load mmc $mmcdev:$mmcpart $loadaddr /boot/zephyr.bin
+   dcache flush; icache flush; go $loadaddr
```

Comment:
Just for your information, we need to disable dcache when use "go" command to boot Zephyr from Core0 on i.MX93, so one solution is to execute "dcache off" before "go", another solution is I submit one fix in https://github.com/zephyrproject-rtos/zephyr/pull/88415, but need to enable some related configuration items when the PR been merged in order to fix this issue in the zephyr booting process.